check-release.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Sanity-checks a release for consistency
+ */
+
+
+
+// get the version reported by phpspec --version
+if (!preg_match_all('/(?<major>[5-9])\.(?<minor>[0-9]+)\.(?<patch>[0-9]+)/', file_get_contents('bin/phpspec'), $matches) || count($matches[0])!=1) {
```

Comment:
why `preg_match_all` rather than `preg_match` when you care only about a single match ?

Reply:
So that it's deliberately brittle if a similar-looking string appears in that file at some point in the future which isn't the version number